README.md

```diff
@@ -257,10 +257,14 @@ The special E2E client settings should be defined in `env.yml`:
 | BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_HOURS | integer | Optional | 24 | Frequency in hours to trigger event requests. |
 | BUGSNAG_PROJECT_NOTIFIER_API_KEY | string | Optional | 123e4567e89b12d3a4564266 | A valid Bugsnag project API key. |
 | BUGSNAG_REPORTING_WINDOW_IN_DAYS | integer | Optional | 14 | Specifies how far in the past events should be retrieved. |
+| BUGSNAG_WEBHOOK_PERMITTED_IPS | string | Optional | 104.196.245.109, 104.196.254.247 | Bugsnag IP addresses which webhook requests are expected to come from. |
+| CONNECTED_DATA_PLATFORM_AGGREGATION_INTERVAL | string | Optional | daily, hourly | Specifies how often to aggregate and upload usage data. Defaults to hourly. |
```

Comment:
Would this be better named reporting interval as referred to on line 265? Or perhaps change line 265 to aggregation interval?

Reply:
I like "reporting" better, thanks for the suggestion. I also renamed "folder aggregation" into "folder grouping" (https://github.com/ibi-group/otp-middleware/pull/254/commits/48e99e954615e61b0d9de1f581ee701eb08b5300).